source/agora/common/Config.d

```diff
@@ -116,6 +119,38 @@ public struct Config
     public EventHandlerConfig event_handlers;
 }
 
+/// Used to specify endpoint-specific configuration
+public struct InterfaceConfig
+{
+    static assert(!hasUnsharedAliasing!(typeof(this)),
+        "Type must be shareable accross threads");
+
+    /// Type of interface one is able to register
+    public enum Type
+    {
+        // FIXME: https://github.com/sociomantic-tsunami/ocean/issues/846
+        // /// Cannonical name is in upper case
```

Comment:
It's because the two lines are commented out. If I used `///` the documentation would incorrectly be applied to the next member. It isn't great but I hope to get this bug fixed in the near-ish future.